profiler/src/Demos/Samples.Computer01/Program.cs

```diff
@@ -23,7 +23,11 @@ public enum Scenario
         Async,
         Iterator,
         GenericsAllocation,
-        ContentionGenerator
+        ContentionGenerator,
+
+#if Linux && NET6_0_OR_GREATER
```

Comment:
Don't #if the enum value: prefer to check in command line parsing 
Otherwise the next values will be different depending on OS and .NET versions

Reply:
✔️